index.js

```diff
@@ -65,33 +101,53 @@ module.exports = {
 
         /* Color */
```

Comment:
May this be changed to `/* Background */`, and move `color` to `Text` section? Cuz most of properties here are not related to `Colors`.

Reply:
It's hard to say where `box-shadow` be placed is the best decision, but I can accept it's placed on current location.

Reply:
`box-shadow` can be placed with `border` & `outline`. In most case, I will set `outline: 0` and use `box-shadow` to customize outline.

Reply:
Agree with @jigsawye 

```
border
box-shadow
outline
```
or
```
border
outline
box-shadow
```

Reply:
```
border
box-shadow
outline
```

👍